### PR TITLE
[SD-2319] - Preview body in published take packages

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

File: client/app/scripts/superdesk-packaging/packaging.js [hunk 1]
```diff
@@ -314,6 +314,10 @@
             link: function(scope) {
                 scope.limits = authoring.limits;
                 scope._editable = scope.origItem._editable;
+                scope._isInProductionStates = scope.origItem.state !== 'published' &&
```
Comment (mdhaman) [inline client/app/scripts/superdesk-packaging/packaging.js]: can we use this 
https://github.com/akintolga/superdesk/blob/updates/client/app/scripts/superdesk-authoring/authoring.js#L273

**Comment by amagdas** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**